.github/workflows/label-accepted.yml

```diff
@@ -5,27 +5,27 @@ on:
     types: [created]
 
 jobs:
-  check_ok_to_test:
+  check_accepted:
     runs-on: ubuntu-latest
-    name: "Check comments for /test"
+    name: 'Check comments for /accept'
     steps:
-      - name: Check for test slash command
+      - name: Check for accept slash command
         uses: xt0rted/slash-command-action@v1
         with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          command: "test"
-          permission-level: "write"
-          reaction: "true"
-          reaction-type: "rocket"
+          repo-token: ${{ secrets.BOT_TOKEN }}
```

Comment:
We now have an official `talos-bot` account, and this is the personal access token for that.